src/Infrastructure/Laravel/Repositories/MysqlVariantRepository.php

```
@@ -23,8 +24,17 @@
 
 class MysqlVariantRepository implements VariantRepository, VariantForCartRepository, StockItemRepository
 {
+    use WithTaxaSelection;
+    use WithVariantKeysSelection;
+
     private static string $productTable = 'trader_products';
     private static string $variantTable = 'trader_product_variants';
+    private static $variantKeysTable = 'trader_product_keys';
+    private static string $taxonProductLookupTable = 'trader_taxa_products';
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$taxonProductLookupTable'. (undefined)

(UnusedPrivateField)